.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 16.14.2
```

Comment:
Maybe here, too? This is for asdf-vm, which has a nice nodejs plugin. I discovered recently they have a decent java plugin that I may start using as well and so the sdkmanrc file may go away unless someone else is using it. Either way, I don't like to tell people how to manage their local environment with tooling since there are so many ways to do it.

Reply:
Same idea as last comment.

If we include these convenience tooling config files in cljdoc we are kinda saying something.

Reply:
But either way, I don't really have a strong opinion, feel free to do what makes sense to you.

Reply:
I'm going to leave it out for now. I'm running a little low on fumes for pushing this feature further. I'll add an issue about it.